src/votekit/utils.py

```diff
@@ -351,23 +402,27 @@ def tiebroken_ranking(
             'first_place'. Defaults to random.
 
     Returns:
-        tuple[frozenset[str], ...]: A list-of-set ranking of candidates (broken down to one
-        candidate sets).
+        tuple[tuple[frozenset[str], ...], dict[frozenset[str], tuple[frozenset[str],...]]]:
+            The first entry of the tuple is a list-of-set ranking of candidates (broken down to one
+            candidate sets). The second entry is a dictionary that maps tied sets to their
+            resolution.
     """
     new_ranking: list[frozenset[str]] = [frozenset()] * len(
         [c for s in ranking for c in s]
     )
 
     i = 0
+    tied_dict = {}
     for s in ranking:
         if len(s) > 1:
             tiebroken = list(tiebreak_set(s, profile, tiebreak))
+            tied_dict[s] = tuple(tiebroken)
```

Comment:
Small nit-pick. Rather than converting to a list and then to a tuple, it would probably be best to use the returned tuple directly (I believe that `tiebreak_set` returns a tuple)

```
tied_dict[s] = tiebreak_set(s, profile, tiebreak)
tiebroken = list(tied_dict[s])
```